src/validation/stringType.ts

```diff
@@ -28,6 +28,15 @@ const FORMAT: {[key: string]: Format} = {
 
         return true;
     },
+    uri: function uri(value: string): boolean {
+        try {
+            // eslint-disable-next-line no-new
+            new URL(value, 'http://any.thing');
```

Comment:
Please add the comment below before this line:
`This simplistic approach covers the most common cases without inflating the library with an RFC 3986-compliant parser.`